test/sequential/test-timers-block-eventloop.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const t1 = setInterval(() => {
+  common.busyLoop(12);
+}, 10);
+
+const t2 = setInterval(() => {
+  common.busyLoop(15);
+}, 10);
+
+const t3 = setTimeout(common.mustNotCall('eventloop blocked!'), 100).unref();
```

Comment:
Nit - the `unref` should not be necessary as the timeout is cleared later on.

Reply:
yes, updated again. :-)